e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction-form.spec.js

```diff
@@ -0,0 +1,146 @@
+
+      cy.assertText(mainHeading(), 'Record correction request');
+    });
+
+    context('when a user has initiated the correction request journey', () => {
+      beforeEach(() => {
+        premiumPaymentsTab.premiumPaymentsTable
+          .checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status)
+          .click();
+        premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+      });
+
+      it('should be able to view the fee record summary on the create record correction request screen', () => {
+        cy.assertText(mainHeading(), 'Record correction request');
+
+        feeRecordSummary.container().should('exist');
+        cy.assertText(feeRecordSummary.facilityId(), feeRecordAtToDoStatus.facilityId);
+        cy.assertText(feeRecordSummary.exporter(), feeRecordAtToDoStatus.exporter);
+        cy.assertText(feeRecordSummary.requestedBy(), `${USERS.PDC_RECONCILE.firstName} ${USERS.PDC_RECONCILE.lastName}`);
+      });
+
+      it('should be able to fill in the form and get redirected to the check the info page', () => {
+        createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT).check();
+        createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.OTHER).check();
+        cy.keyboardInput(createFeeRecordCorrectionRequestPage.additionalInfoInput(), additionalInfoUserInput);
+
+        cy.clickContinueButton();
+
+        cy.assertText(mainHeading(), 'Check the information before submitting the record correction request');
+      });
+    });
+
+    context('when user clicks back on the create record correction request screen', () => {
+      it('should return to premium payments tab with the checkbox selected', () => {
+        premiumPaymentsTab.premiumPaymentsTable
+          .checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status)
+          .click();
+
+        premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+
+        cy.clickBackLink();
+
+        cy.url().should('eq', relative(`/utilisation-reports/${reportId}?selectedFeeRecordIds=${feeRecordAtToDoStatus.id}`));
+
+        premiumPaymentsTab.premiumPaymentsTable
+          .checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status)
+          .should('be.checked');
+      });
+    });
+
+    it('should let the user enter additional info equal to the character limit containing special characters', () => {
+      premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
+
+      premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+
+      createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.OTHER).check();
+
+      const specialCharactersToTest = '&!?$£¥€¢^*()_+=-%:;@~/><,.';
```

Comment:
is it possible to check that these characters are displayed when coming back to the page.
If handling for special characters breaks for this page, we will be able to catch it in a test

Reply:
I will add some special characters to the additional info in the check the info page tests